service-api/src/test/java/greencity/exception/exceptions/InvalidEmailExceptionTest.java

```diff
@@ -0,0 +1,22 @@
+package greencity.exception.exceptions;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class InvalidEmailExceptionTest {
+
+    @Test
+    void testMessageConstructor() {
+        String message = "Custom exception message";
+        InvalidEmailException exception = Mockito.spy(new InvalidEmailException(message));
+        String actualMessage = exception.getMessage();
+        assertEquals(message, actualMessage);
+        verify(exception).getMessage();
```

Comment:
There is not a reason to test this, if you wanna validate the message, please validate it in the service where this method is used.